srcpkgs/chroot-curl/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'chroot-curl'
+pkgname=chroot-curl
+version=7.79.1
+revision=1
+bootstrap=yes
+wrksrc=curl-${version}
+build_style=gnu-configure
+configure_args="--enable-threaded-resolver --enable-ipv6 --with-openssl
+ --with-ca-bundle=/etc/ssl/certs/ca-certificates.crt ac_cv_sizeof_off_t=8
+ --disable-shared"
+hostmakedepends="perl pkg-config"
+makedepends="openssl-devel zlib-devel"
+depends="ca-certificates"
+short_desc="Client that groks URLs - chroot package"
+maintainer="Orphaned <orphan@voidlinux.org>"
```

Comment:
This can be assigned to me.  Ultimately I'd like to have infrastructure@ own things like this, but that can come later.